scr/main/java/org/micks/champmaker/matches/CreateScheduleRequest.java

```diff
@@ -14,4 +18,16 @@ public CreateScheduleRequest() {
     public Long getChampId() {
         return champId;
     }
+
+    public void setChampId(Long champId) {
```

Comment:
If setter for champId was not required before, then looks like we don't need setters at all. So let's remove this one and setter for List<Object> too.

Reply:
fixed